src/frontend/src/btc/components/transactions/BtcTransaction.svelte

```diff
@@ -25,4 +28,5 @@
 	{type}
 	{timestamp}
 	{pending}
+	{unconfirmed}
```

Comment:
Can it be pending and unconfirmed at the same time? If not, would it be safer to pass the status here?

Reply:
True, we can just pass `status`. I did it like this now, it lead to a small refactoring of the types, please take a look.